test/integration/iam-identity.v1.test.js

```diff
@@ -1565,6 +1589,521 @@ describe('IamIdentityV1_integration', () => {
     expect(response).not.toBeNull();
     expect(response.iam_id).not.toBeNull();
   });
+
+  test('scenarioProfileTemplate()', async () => {
+    await createProfileTemplate();
```

Comment:
This seems like an odd way to organize the testcases, with a single test() block that triggers a chained invocation of the tests for the various operations associated with this resource type.  Please refactor these new tests so they follow the same pattern as the existing integration tests - one test() block per operation.

Reply:
Hi Phil, I first had it this way but then the wait for assignment to be finished didn't work. I'm very happy I got it working that way. If you want it changed just for style reasons please show me how.